trunk/SUAVE/Methods/Aerodynamics/AVL/Data/Cases.py

```diff
@@ -72,45 +72,29 @@ def __defaults__(self):
         free.gravitational_acceleration = 9.81
 
         aero.parasite_drag              = 0.0
-        aero.angle_of_attack            = 0.0
+        aero.angle_of_attack            = None
+        aero.flight_CL                  = None
         aero.side_slip_angle            = 0.0
 
-        self.stability_and_control.control_deflections  = None
+        self.stability_and_control.control_surface_names     = None
+        self.stability_and_control.control_surface_functions = None
         self.stability_and_control.number_control_surfaces = 0
         self.conditions.freestream      = free
         self.conditions.aerodynamics    = aero
 
-        self.result_filename            = None
-        self.eigen_result_filename      = None
- 
-
-    def append_control_deflection(self,control_tag,deflection):
-        """ Adds a control deflection case 
-
-	Assumptions:
-	    None
-    
-	Source:
-	    None
-    
-	Inputs:
-	    None
-    
-	Outputs:
-	    None
-    
-	Properties Used:
-	    N/A
-	"""         
-        control_deflection              = Data()
-        control_deflection.tag          = control_tag
-        control_deflection.deflection   = deflection
-        if self.stability_and_control.control_deflections is None:
-            self.stability_and_control.control_deflections = Data()
-        self.stability_and_control.control_deflections.append(control_deflection)
-
+        self.aero_result_filename_1     = None
+        self.aero_result_filename_2     = None
+        self.aero_result_filename_3     = None
+        self.aero_result_filename_4     = None
+        self.aero_result_filename_5     = None
+        self.aero_result_filename_6     = None
+        self.aero_result_filename_7     = None
+        self.aero_result_filename_8     = None 
+        self.aero_result_filename_9     = None      
+        self.eigen_result_filename_1    = None 
+        self.eigen_result_filename_2    = None 
         return
-
+ 
 class Container(DataOrdered):
     """ A data class for the addition of a cases to the set of run cases
```

Comment:
update doc string

Reply:
Updated it but I did a search for places where there is " A class" and there are in more places than the ones I changed. Like its like 40/60 throughout so we either need to change all or not be as stringent with the style because it can be confusing